chapter2/tests/t2/chapter2-t2-gen-kustomization.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+if [ "$#" -eq 0 ]; then
+  echo "Usage: chapter2-t2-gen-kustomization.sh [REPO]..."
+  exit 1
+fi
+
+readonly DEPLOY_KEY_POSTFIX="deploy-key.pem"
+
+# stop on first error
+set -e
+
+# init, prepare kustomize
+mkdir -p repos
+echo \
+"apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+generatorOptions:
+  disableNameSuffixHash: true
+
+secretGenerator:"  > ./repos/kustomization.yaml
+
+# iter thru repos
+while (( $# )); do
+  repo=$(basename $1)
+  deploy_key="./repos/$repo-$DEPLOY_KEY_POSTFIX"
+  
+  # create deploy key
+  rm -f $deploy_key
+  ssh-keygen -t ed25519 -f $deploy_key -N '' -m pem -q
```

Comment:
You can do like this instead of remove existing key above to overwrite it (to have 1 line instead of 2):
```
ssh-keygen -t ed25519 -f $deploy_key -N '' -m pem -q <<< y >/dev/null 2>&1
```